x-pack/plugins/apm/public/tutorial/tutorial_fleet_instructions/tutorial_fleet_instructions.stories.tsx

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+/* eslint-disable react/function-component-definition */
+
+import { Story } from '@storybook/react';
+import React from 'react';
+import { HttpStart } from 'kibana/public';
+import TutorialFleetInstructions from '.';
+
+interface Args {
+  hasFleetPoliciesWithApmIntegration: boolean;
+}
+
+function Wrapper({ hasFleetPoliciesWithApmIntegration }: Args) {
+  const http = ({
+    get: () => ({ hasData: hasFleetPoliciesWithApmIntegration }),
+  } as unknown) as HttpStart;
+  return (
+    <TutorialFleetInstructions
+      http={http}
+      basePath="http://localhost:5601"
+      isDarkTheme={false}
+    />
+  );
+}
+
+export default {
+  title: 'app/Tutorial/FleetInstructions',
+  component: TutorialFleetInstructions,
+  argTypes: {
+    hasFleetPoliciesWithApmIntegration: {
+      control: { type: 'boolean', options: [true, false] },
+    },
+  },
+};
+
+export const Instructions: Story<Args> = (_args) => {
```

Comment:
Starting an argument with `_` is usually only done when the argument isn't used, by convention and for TypeScript, so this can just be `args`.